openssl-1.0.1f/target.cc

```diff
@@ -21,9 +30,11 @@ SSL_CTX *Init() {
       openssl req -x509 -newkey rsa:512 -keyout server.key \
      -out server.pem -days 9999 -nodes -subj /CN=a/
   */
-  assert(SSL_CTX_use_certificate_file(sctx, CERT_PATH "server.pem",
+  assert(SSL_CTX_use_certificate_file(sctx,
+                                      GetPath("runtime/server.pem").c_str(),
```

Comment:
why can we make path same during testing and build?

Reply:
The benchmark is built on the dispatcher VM, and the binary is copied to a runner VM.  The path on the runner VM is different than on the dispatcher VM.

So we could either create a  bunch of folders just to hold the cert/key, or we can check at runtime if the files exist and otherwise expect them in the `runtime/` folder.

Reply:
I also prefer this approach in case there are future benchmarks requiring files at runtime.

Reply:
Can we just always copy cert next to the binary?

Reply:
Yes, this should be possible and a cleaner solution.  I think I can modify build.sh to do that, so we avoid breaking the tutorial.